operations/helm/scripts/build.sh

```diff
@@ -72,13 +75,34 @@ for FILEPATH in $TESTS; do
     ARGS+=("--set-string" "kubeVersionOverride=${DEFAULT_KUBE_VERSION}")
   fi
 
-  set -x
-  helm template "${ARGS[@]}"
-  set +x
+  helm template "${ARGS[@]}" 1>/dev/null &
+  pid=$!
+  pids+=("$pid")
+  echo "Launched helm template PID $pid 'helm template ${ARGS[*]}'"
+done
+
+# Wait for all child processes to finish. We turn off `set -e` because we want to check the exit code of each child process
+# and print a message if it's non-zero. set -e will otherwise cause the script to exit immediately if any child process has exited with non-zero exit code.
+set +e
+for p in "${pids[@]}"; do
+    wait "$p"
+    exit_code=$?
+    if [ $exit_code -ne 0 ]; then
+        echo "helm template PID $p exited with non-zero exit code $exit_code. Aborting."
+        exit $exit_code
+    fi
+done
+set -e
```

Comment:
Two questions on this:

* If multiple `helm template` invocations fail, is their output clearly separated? Or will they all try to write to stdout/stderr at the same time and the output will be interleaved?
* Should we wait for all `helm template` invocations to complete, even if one fails, then exit if any failed? Otherwise later invocations might continue running in the background and produce output at unexpected times (when running locally) or be lost altogether (when running on CI).